developers/deployments/addresses.md

```diff
@@ -37,9 +37,11 @@
 
-\*BridgedUSDC is a Mock ERC20 token that we use on testnets as a surrogate for the bridged USDC that will be used in pools Celo Mainnet.
+\*Bridged USDC/EUROC are Mock ERC20 tokens that we use on testnets as a surrogate for the bridged USDC/EUROC that will be used in pools Celo Mainnet.
+
+\*mockNativeUSDC is a Mock ERC20 token that we use on Baklava as a surrogate for the native USDC that will be used in pools and is only deployed on Alfajores and Mainnet.
```

Comment:
Is this sentence correct? "...we use on Baklava...and is only deployed on Alfajores and Mainnet." ?